packages/span-transformations/README.md

```diff
@@ -0,0 +1,12 @@
+# `span-transformations`
```

Comment:
I could but I don't see the value, can you explain what we gain from separating?

Reply:
Each PR should ideally address one thing. The reasons are:
1. Your PR title is now "feat: Upgrade otel instrumentation version to 0.27". That means that the changelog for the new span transformations package will contain wrong description. It will show up in the changelog and release notes as if it was part of upgrading to v0.27.0 which is incorrect and confusing.
2. It is harder to review large PRs, especially when they mix multiple features together. It is also easier for bugs to slip inside when touching large parts of the codebase at once.
3. If we found a bug and need to revert the PR, we now need to revert 2 features instead of just the feature with the issue.
4. Creating PRs with multiple features is considered bad practice in the industry. PRs that address one issue is a signal of code quality and professional work which is what we want to reflect in our public work.

Reply:
Ok i'll separate after all issues resolved thanks